patternlab/styleguide/source/_patterns/02-molecules/location-filters.twig

```diff
@@ -34,9 +34,15 @@
       </div>
     {% endif %}
     <div class="ma__location-filters__buttons">
-      <button class="ma__location-filters__submit" type="submit">
+      {% for button in locationFilters.buttons %}

```

Comment:
please verify this change doesn't have unexpected impact in drupal

Reply:
In Drupal, currently only the submit button is implemented for location listing.  I don't think there is any impact in Drupal with this change.
Also, event listing page has no filter implemented.  

<img width="715" alt="_qa_full_test_organization_locations___mass_gov_1" src="https://user-images.githubusercontent.com/9633303/44485960-444f1d80-a620-11e8-9b81-7796b8a2f825.png">